### ISSUE TITLE
Hover over icon and see stats

### ISSUE THREAD

**Issue by SanaNoHana** [KC3Kai/KC3Kai#865]
I see we can see FP, Torp, AA, and Armor on abyssals when we hover over their icons. Why can't we do the same for our own ships? :)

**Comment by LostLogia4** [comment]
Do you want it to be with equips, or without?

**Comment by SanaNoHana** [comment]
Does the abyssal one show with their equipment?

**Comment by bamboo3250** [comment]
Regardless of abyssal side, it still makes more sense to me to show stats with equipment in the battle.

**Comment by LostLogia4** [comment]
@SanaNoHana iinm it didn't.

**Comment by SanaNoHana** [comment]
Probably would be better to show with equipment then, both for Abyssals and Kanmasu

**Comment by dragonjet** [comment]
related #840
might be when you click on a ship, #840 shows up?

**Comment by LostLogia4** [comment]
@dragonjet This is gonna be rather distracting during sorties...

**Comment by Yukinyaa** [comment]
isn't this implemented?

**Comment by sinsinpub** [comment]
yes partially.

but men maybe wants more such as:
 * More details to show up
 * More pretty popup layer than system tooltip
 * Additional fire power by improvement or proficiency

My attempt is just show up the data already provided by game or our database, without more calculating or summarizing.
For example: the battle node API provides abyssal (or PvP opponent) ship's 4 stats without equipment's additions. To get values with equips, have to summarize from all the equips ship is carrying. Ofc other stats beside that 4 are not able to show...
And another example: for our shipgirls, the stats values already include the additions of equips. But to show up useful values for battle, have to do more calculations on the akashi improved equips or proficiency of planes.

So basically for now, using well-maintained calculator, such as 'Kancolle-calc DeckBuilder', is better way... :sweat_smile:

**Comment by dragonjet** [comment]
I believe we already have this?

**Comment by LostLogia4** [comment]
We already have 1 and 2, but we don't have firepower bonus just yet.

**Comment by sinsinpub** [comment]
@LostLogia4 wait, i just meant stats of our ships as an example... for enemy (abyssal/pvp), it's impossible to get improvement bonus from api 😂  so for this topic only, i think we already have all.
